lstchain/scripts/longterm_dl1_check.py

```diff
@@ -1,47 +1,84 @@
+whole run, with name pattern datacheck_dl1_LST-1.Run?????.h5. It takes all
+files in the input directory. It also reads, if available in the same path,
+the corresponding muons*fits files.
+
+The output is the file longterm_dl1_check.h5 file (the name can be modified via
+commandline), which contains tables with some run-wise summary values for
+plotting long-term evolution of the DL1 data.
 It also produces an interactive web page, longterm_dl1_check.html with plots
-showing the evolution of many such values.
+showing the evolution of many such values. If not in batch mode, the page is
+opened by the default browser at the end of execution.
+
+It also produces a longterm_dl1_check.log file with warnings about values
+which are beyond certain limits.
 
 """
 
+import argparse
+import logging
 import copy
-import glob
 from pathlib import Path
 
 import numpy as np
 import pandas as pd
 import tables
 from astropy.table import Table
 from bokeh.io import output_file as bokeh_output_file
-from bokeh.io import show
+from bokeh.io import show, save
 from bokeh.layouts import gridplot, column
 from bokeh.models import Div, ColumnDataSource, Whisker, HoverTool, Range1d
 from bokeh.models.widgets import Tabs, Panel
 from bokeh.plotting import figure
 from ctapipe.coordinates import EngineeringCameraFrame
 from ctapipe.instrument import SubarrayDescription
+from ctapipe_io_lst import TriggerBits
+
+from lstchain.visualization.bokeh import show_camera, get_pixel_location
+
+log = logging.getLogger(__name__)
+
+parser = argparse.ArgumentParser(description="DL1 multi-run data checker")
+parser.add_argument('--input-dir', '-d', action='store', type=Path,
+                    dest='input_dir',
+                    help='path to the datacheck_dl1_LST-1.RunXXXXX.h5 files',
+                    default='./')
 
-from lstchain.visualization.bokeh import show_camera
+parser.add_argument('--output-file', '-o', action='store', type=Path,
+                    dest='output_file',
+                    help='.h5 output file name',
+                    default='./longterm_dl1_check.h5')
 
+parser.add_argument('--batch', '-b', action='store_true',
+                    help='Run the script without opening html output'
+                    )
 
 def main():
 
-    output_file_name = 'longterm_dl1_check.h5'
-    files = glob.glob('datacheck_dl1_LST-1.Run?????.h5')
+    args = parser.parse_args()
+
+    output_file_name = args.output_file
+    # Create directories if needed:
+    output_file_name.parent.mkdir(parents=True, exist_ok=True)
+
+    log.setLevel(logging.INFO)
+    logfilename = output_file_name.with_suffix('.log')
+    handler = logging.FileHandler(logfilename, mode='w')
+    logging.getLogger().addHandler(handler)
+
+    files = sorted(args.input_dir.glob('datacheck_dl1_LST-1.Run?????.h5'))
+
     if not files:
         raise IOError("No input datacheck files found")
-    files.sort()
+
+    print('Processing', len(files), 'datacheck files...')
```

Comment:
Mixed `print` with `logging` to a file throughout the rest of the script. Maybe this printout could be set through the `StreamHandler` or directly written to the `.log` file.

Reply:
I do not want to log this into the file, but show it in stdout just for the user to know what is going on when run interactively.